### PR TITLE
Make brunel.py setup faster.

### PR DESCRIPTION
**Pull request by thorstenhater**
Small tweak, gets 10% setup time reduction for 500 cells. Will be more w/ larger networks.

### CODE REVIEW

File: python/example/brunel.py [hunk 1]
```diff
@@ -21,13 +21,14 @@
...
-    s = set()
-    while len(s) < m:
-        val = gen.randint(low=start,high=end)
-        if val != gid:
-            s.add(val)
-    return s
+    end = min(end, start + m)
+    idx = numpy.arange(start, end + 1)
+    if start < gid < end:
+        numpy.delete(idx, gid - start)
```
Comment (brenthuisman) [inline python/example/brunel.py:27]:
```suggestion
        idx = numpy.delete(idx, gid - start)
```
`.delete()` is not in-place. Also, due to the `+1` above, `end` will occur in the list, right? The function description says it returns on interval `[start,end)`.

File: python/example/brunel.py [hunk 2]
```diff
@@ -21,13 +21,14 @@
...
+    else:
+        numpy.delete(idx, end - start)
```
Comment (brenthuisman) [inline python/example/brunel.py:29]:
```suggestion
        idx = numpy.delete(idx, end - start)
```

**Review by brenthuisman — CHANGES_REQUESTED** [review]
Not too sure about including `end` in case idx is in range.

File: python/example/brunel.py (hunk 1 above)
Comment (thorstenhater) [reply python/example/brunel.py:27]: Yes. We over-allocate one item and delete one. Makes net zero.

Comment (brenthuisman) [reply python/example/brunel.py:27]: Length, yes, but at the cost of returning values in interval `[start, end]`, no? Maybe it's not a problem?

Comment (thorstenhater) [reply python/example/brunel.py:27]: As stands this is completly busted. It'll return something in range `[start, start + m)`.

**Review by brenthuisman — APPROVED** [review]

**Pull request merged**